src/pages/index.js

```diff
@@ -99,7 +99,7 @@ const IndexPage = ({ data }) => {
                     <ul className="three-col-md">
                       {taggedPagesPubbed.map((taggedPage, index) => (
                         <li key={`tagged-${index}`}>
-                          <Link to={taggedPage.path.alias}>{taggedPage.title}</Link>
+                          <Link to={taggedPage.path.alias.toLowerCase()}>{taggedPage.title}</Link>
```

Comment:
I think this might still work without switching it to lower-case (I believe Netlify automatically redirects it...but I get that by us adding the switch to lower-case, we are avoiding the extra redirect)